ijs-Run-TrackMate-Using-RoiManager-or-ResultsTable.groovy

```diff
@@ -0,0 +1,573 @@
+ * 
+ * 
+ * == OUTPUTS ==
+ * The Tracking results from Trackmate through the TrackMate GUI for manual curation of the tracks if desired.
+ * It also saves both the TrackMate project file and the tracks as XML files.
+ *
+ * = DEPENDENCIES =
+ * FIJI ( which bundles TrackMate )
+ * Tested on TrackMate 
+ * 
+ * = INSTALLATION = 
+ * Open the script in FIJI and hit run
+ * 
+ * = AUTHOR INFORMATION =
+ * Orignal Python code by Eugene Katrukha: https://gist.github.com/ekatrukha/ac32d624665da81ad521ab4b040a4bad
+ * Code Groovyfied and extended by Olivier Burri, EPFL - SV - PTECH - BIOP
+ * Reviewing, Testing by Romain Guiet, EPFL - SV - PTECH - BIOP 
+ * February 2021
+ * 
+ * = COPYRIGHT =
+ * © All rights reserved. ECOLE POLYTECHNIQUE FEDERALE DE LAUSANNE, Switzerland, BioImaging And Optics Platform (BIOP), 2021
+ * 
+ * Licensed under the BSD-3-Clause License:
+ * Redistribution and use in source and binary forms, with or without modification, are permitted provided 
+ * that the following conditions are met:
+ * 1. Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer 
+ *    in the documentation and/or other materials provided with the distribution.
+ * 3. Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote products 
+ *     derived from this software without specific prior written permission.
+ * 
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, 
+ * BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. 
+ * IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, 
+ * EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; 
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, 
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF 
+ * ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*/ 
+
+//-----------------------------
+//			MAIN 
+//-----------------------------
+
+// Get ROIs from the Manager
+def roiList = rm.getRoisAsArray() as List
+
+// Example: Manipulate ROIS to change the position here if needed here
+// EG: Here we switch the T position of a roi to pat to the "regular" position
+//def roiList = roiList.collect{ roi -> 
+//	def p = roi.getTPosition()
+//	roi.setPosition( p )
+//	return roi
+//}
+
+
+// Example: Get results table if the spots should be populated by a results table
+//def resultsTable = ResultsTable.getResultsTable()
```

Comment:
So are these comments necessary or should they be removed ?

Reply:
input can be ROIs in the manager or a ResultTable with all information needed